app/assets/javascripts/discourse/app/components/sidebar/section.js

```diff
@@ -4,11 +4,27 @@ import { action } from "@ember/object";
 import { tracked } from "@glimmer/tracking";
 
 export default class SidebarSection extends GlimmerComponent {
-  @tracked displaySection = true;
+  @tracked displaySection;
+  collapsedSidebarSectionKey = `sidebar-section-${this.args.sectionName}-collapsed`;
+
+  constructor() {
+    super(...arguments);
+
+    this.displaySection =
+      this.keyValueStore.getItem(this.collapsedSidebarSectionKey) === undefined
+        ? true
+        : false;
```

Comment:
Do you need the ternary here? The `===` should always return `true` or `false`, right?